gsuite/utils.go

```diff
@@ -46,6 +47,17 @@ func retryTime(retryFunc func() error, minutes int, retryNotFound bool) error {
 				return resource.RetryableError(gerr)
 			}
 		}
+
+		// Deal with the broken API
+		if strings.Contains(fmt.Sprintf("%s", err), "Invalid Input: Bad request for ") && strings.Contains(fmt.Sprintf("%s", err), "\"code\":400") {
```

Comment:
this is absolutely ridiculous, but this is how I'm going around the broken API:
- create user without POSIX
- make sure the user exists
- extend user with POSIX data
- the API will return this vague error for the first half minute
- at some point we will either have the POSIX data set or it will fail on the service unavailable 503 which can occur when POSIX data is not unique
- not failing on-creation when setting POSIX fails, otherwise we don't have the data in terraform and can't manage it at all